src/soorgeon/clean.py

```diff
@@ -32,22 +32,24 @@ def lint(task_file):
 
 
 @telemetry.log_call('clean')
-def basic_clean(task_file, program="black"):
+def basic_clean(task_file, program="black", string_normalization=True):
     """
     Run basic clean (directly called by cli.clean())
     Generate intermediate files for ipynb
     """
     with get_file(task_file, write=True) as path:
-        clean_py(path, task_file)
+        clean_py(path, task_file, string_normalization=string_normalization)
 
     click.echo(f"Finished cleaning {task_file}")
 
 
-def clean_py(task_file_py, filename):
+def clean_py(task_file_py, filename, string_normalization=True):
+    mode = FileMode(string_normalization=string_normalization)
```

Comment:
why is this string_normalization parameter needed?

Reply:
When `string_normalization=False`, black will not substitute single quotes with double quotes (AKA do not mess with quotes).
It is only used when we are automatically applying black before refactoring, it does not change the API for this function & allows for backward compatibility.

But why do we need to keep single quotes ? 
2 reasons:
1. Correct me if I'm wrong, but I think not all users will be happy if we substitute the single quotes.
2. Some tests related to refactoring, started to fail when black was substituting single quotes with double quotes.  Turns out parts of the code use regex to search for single quotes.

Reply:
makes sense. thanks for the clarification!